test/powershell/Modules/Microsoft.PowerShell.Utility/assets/invalid_node.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "James",
+  "hobbies": [".NET", "Blogging", "Reading", "Xbox", "LOLCATS"]
+  errorNode
+}
```

Comment:
I'd prefer to generate the assert files on the fly. It is more readable.

Reply:
Using Pester `TestDrive:/` instead to generate on the fly.